packages/ERTP/src/types.js

```diff
@@ -238,7 +238,10 @@
  * @property {() => Brand} getAllegedBrand Get the alleged Brand for this Purse
  *
  * @property {() => Amount} getCurrentAmount
- * Get the amount contained in this purse, confirmed by the issuer.
+ * Get the amount contained in this purse. For a valid purse, the amount
+ * reported will be consistent with the Issuer's records, but since the message
+ * is sent directly to the purse, a fake purse could report a balance that
+ * doesn't match the Issuer's balance, and so can't be spent.
```

Comment:
Should we add something about a fake purse being unlikely / impossible if the right issuer made the purse?

Reply:
I wonder if we should just remove `confirmed by the issuer` and not add anything additionally.